.pullapprove.yaml

```diff
@@ -0,0 +1,34 @@
+version: 2
+
+requirements:
+  signed_off_by:
+    required: true
+
+always_pending:
+  title_regex: '(WIP|wip)'
+  explanation: 'Work in progress...'
+
+group_defaults:
+  approve_by_comment:
+    enabled: true
+    approve_regex: '^(Approved|LGTM|:shipit:|:\+1:|Engage)'
+    reject_regex: '^(Rejected|:-1:|Borg)'
+  reset_on_push:
+    enabled: true
+  reset_on_reopened:
+    enabled: true
+  author_approval:
+    ignored: true
```

Comment:
http://docs.pullapprove.com/groups/author_approval/
I'm not sure if we want to use ignored or required. Required is IMO the best option, but we would have to tell every contributer that he should approve his own PR. Which no repo I know does.